dotnet/src/SemanticKernel.Abstractions/Orchestration/SKContext.cs

```diff
@@ -63,7 +63,7 @@ public CultureInfo Culture
     /// <summary>
     /// AI service provider
     /// </summary>
-    internal IAIServiceProvider ServiceProvider { get; }
+    public IAIServiceProvider ServiceProvider { get; }
```

Comment:
Not sure why we need to expose the ServiceProvider, I would rather expose the Selector

Reply:
Changed this because we now pass the `SKContext` to the `IAIServiceSelector`. This change will align with the work that Sergey and Stephen are doing with refactoring. This will eventually become a `Kernel` instance and I can call `GetService` from there.